### PR TITLE
MVC, service, H2 tests, Postman responces

### PR DESCRIPTION
**Pull request by Artsapollo**

### CODE REVIEW

**Comment by Artsapollo** [comment]
There are three responce files from Postman in the end

File: src/main/java/springData/controller/OfficeController.java [hunk 1]
```diff
@@ -16,13 +20,47 @@
     @Autowired
     private OfficesService officesService;
 
+    @GetMapping
+    public @ResponseBody
+    Set<Office> findOfficeByTargetBetween(@RequestParam(value = "min", required = false) Integer minTarget,
+                                          @RequestParam(value = "max", required = false) Integer maxTarget) {
+        LOG.info("findByTargetBetween start, min={}, max={}", minTarget, maxTarget);
+        if (Objects.isNull(maxTarget) || Objects.isNull(minTarget)) {
```
Comment (un-install) [inline src/main/java/springData/controller/OfficeController.java]: wrong logic. you can return getAllOrders only witn [and] statement

File: src/main/java/springData/controller/OfficeController.java [hunk 2]
```diff
@@ -16,13 +20,47 @@
...
-        Office result = officesService.findOfficeById(java.math.BigDecimal.valueOf(id));
+        Office result = officesService.findOfficeById(BigDecimal.valueOf(id));
         LOG.info("findOfficeById end");
-        throw new DeleteException("ERROR");
-//        return result;
+        return result;
+    }
+
+    @PutMapping("/{id}")
+    public void updateOfficeById(@PathVariable("id") Integer id, @RequestParam("city") String city) {
```
Comment (un-install) [inline src/main/java/springData/controller/OfficeController.java]: Thats no good, that you can only update one field of entity. Maybe you should try to use json request?

File: src/main/java/springData/service/OfficesServiceImpl.java [hunk 3]
```diff
@@ -34,45 +43,35 @@
 
     @Override
     public Office findOfficeById(BigDecimal id) {
-        LOG.debug("getOfficeByCity, city={}", id);
+        LOG.debug("findOfficeByCity, city={}", id);
         Office result = officesRepository.findById(id).orElse(null);
```
Comment (un-install) [inline src/main/java/springData/service/OfficesServiceImpl.java]: bad using of optional

**Pull request merged**